### PR TITLE
Lesson about Python Programming Paradigms

### PR DESCRIPTION
**Pull request by Pyrotheum1702**

### CODE REVIEW

File: 15-python-programming-paradigms/README.MD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - OOP most important concept:
+      - Inheritance
+      - Polymorphism
+      - Encapsulation
+      - Abstraction
+  - Procedural Programming:
+    - Procedural programming mean to program in series of computational steps, the execution instructions that happen in step by step to reach a goal or to complete a certain task.
+    - Procedural programming paradigm focuses on the process rather than data.
+    - Procedural programming is best for simple programs but is better is use other programming paradigms for solving complex programs.
+  - Functional programming:
```
[the remark this answers is not in the document]
Comment (Pyrotheum1702) [reply 15-python-programming-paradigms/README.MD:25]: Added pros and cons but it's all seem just a theory to me. Could'nt imagine how it is.

File: 15-python-programming-paradigms/README.MD [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      - Code reusability.
+      - Abstraction or data hiding.
+    - Cons:
+      - Data protection.
+      - Not suitable for all types of problems.
+      - Slow Speed.
+  - Procedural Programming:
+    - Procedural programming mean to program in series of computational steps, the execution instructions that happen in step by step to reach a goal or to complete a certain task.
+    - Procedural programming paradigm focuses on the process rather than data.
+    - Procedural programming is best for simple programs but is better is use other programming paradigms for solving complex programs.
```
Comment (vanhiepdam) [inline 15-python-programming-paradigms/README.MD:32]: not correct. FP and OOP is just a style

File: 15-python-programming-paradigms/README.MD [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      - Inheritance.
+      - Polymorphism.
+      - Encapsulation.
+      - Abstraction.
+    - Pros:
+      - Relation with Real world entities.
+      - Code reusability.
+      - Abstraction or data hiding.
+    - Cons:
+      - Data protection.
```
Comment (vanhiepdam) [inline 15-python-programming-paradigms/README.MD:26]: OOP has so many problems than these. find out more

File: 15-python-programming-paradigms/README.MD [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    - The most importance concept in functional programming is the concept of pure function.
+      - Mean the output of the function should only depend on it's input or arguments.
+    - Pros:
+      - Simple to understand.
+      - Making debugging and testing easier.
+      - Enhances the comprehension and readability of the code.
+    - Cons:
+      - Low performance.
+      - Writing programs is a daunting task.
+      - Low readability of the code.
```
[the remark this answers is not in the document]
Comment (vanhiepdam) [reply 15-python-programming-paradigms/README.MD:52]: https://viblo.asia/p/tam-biet-object-oriented-programming-3Q75wBBMlWb

**Review by vanhiepdam — APPROVED** [review]

**Pull request merged**